### PR TITLE
added models service

### PR DESCRIPTION
**Pull request by BelehovEgor**

### CODE REVIEW

**Review by MariyaSin — APPROVED** [review]

File: src/MessageService.Models.Dto/Configurations/RabbitMqConfig.cs [hunk 1]
```diff
@@ -1,11 +1,19 @@
...
+﻿using LT.DigitalOffice.Kernel.Attributes;
+using LT.DigitalOffice.Kernel.Configurations;
+using LT.DigitalOffice.Models.Broker.Requests.File;
+using LT.DigitalOffice.Models.Broker.Requests.Message;
 
 namespace LT.DigitalOffice.MessageService.Models.Dto.Configurations
 {
     public class RabbitMqConfig : BaseRabbitMqConfig
     {
+        [AutoInjectRequest(typeof(ISendEmailRequest))]
```
Comment (djino97) [inline src/MessageService.Models.Dto/Configurations/RabbitMqConfig.cs:10]: not need attribute

File: src/MessageService.Models.Dto/Configurations/RabbitMqConfig.cs [hunk 2]
```diff
@@ -1,11 +1,19 @@
...
         public string SendEmailEndpoint { get; set; }
-        public string CreateImageEndpoint { get; set; }
+
+        [AutoInjectRequest(typeof(IAddImageRequest))]
+        public string AddImageEndpoint { get; set; }
+
+        [AutoInjectRequest(typeof(IGetEmailTemplateTagsRequest))]
```
Comment (djino97) [inline src/MessageService.Models.Dto/Configurations/RabbitMqConfig.cs:16]: not need attribute

**Review by djino97 — APPROVED** [review]

File: src/MessageService.Models.Dto/Configurations/RabbitMqConfig.cs (hunk 1 above)
Comment (BelehovEgor) [reply src/MessageService.Models.Dto/Configurations/RabbitMqConfig.cs]: fixed

File: src/MessageService.Models.Dto/Configurations/RabbitMqConfig.cs (hunk 2 above)
Comment (BelehovEgor) [reply src/MessageService.Models.Dto/Configurations/RabbitMqConfig.cs]: fixed

**Pull request merged**